reports/crash_stats_oom/getting-crash-stats-for-OOM-data-to-S3.ipynb

```diff
@@ -54,7 +55,29 @@
     "    \"\"\" Return the int representation of obj, or None \"\"\"\n",
     "    if obj is None:\n",
     "        return None\n",
-    "    return int(obj)\n"
+    "    return int(obj)\n",
+    "\n",
+    "\n",
+    "def get_token_from_s3():\n",
+    "    \"\"\" Get socorro token from data already saved to s3 \"\"\"\n",
+    "    data_bucket = \"net-mozaws-prod-us-west-2-pipeline-analysis\"\n",
+    "    s3path = \"ddurst/oom\"\n",
+    "    filename = \"oom.tkn\"\n",
+    "    key = \"{}/{}\".format(s3path, filename)\n",
+    "\n",
+    "    s3 = boto3.resource('s3')\n",
+    "\n",
+    "    try:\n",
+    "        obj = s3.Object(data_bucket, key)\n",
+    "    except botocore.exceptions.ClientError as e:\n",
+    "        # If the file wasn't there, that's ok. Otherwise, abort!\n",
+    "        if e.response['Error']['Code'] != \"404\":\n",
```

Comment:
Seems like a 404 here is not actually ok - you need this token later. Depending on how you want to handle a missing / unavailable token, it might be best to just remove the `try` block and let the code raise an error if the token can't be fetched.